src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -133,6 +133,22 @@ enum ObjectiveCType {
   OBJECTIVECTYPE_MESSAGE
 };
 
+template<class TDescriptor>
+string GetOptionalDeprecatedAttribute(const TDescriptor* descriptor, bool preSpace = true, bool postSpace = false) {
```

Comment:
Yes please, that way the support lands as one atomic unit just incase there's an issue.